resources/lib/cabertoss.py

```
@@ -109,11 +118,12 @@ def copy_log_files(log_files: []):
         for file in log_files:
             if file[0] in ['log', 'oldlog']:
                 Logger.info(f'Copying sanitised {file[0]} {file[1]}')
-                with open(xbmcvfs.translatePath(file[1]), 'r', encoding='utf-8') as current:
+                with open(xbmcvfs.translatePath(file[1]), 'r', encoding='utf-8', errors='replace') as current:
                     content = current.read()
                     sanitised = clean_log(content)
```

Ruff (0.13.1)

`clean_log` may be undefined, or defined from star imports

(F405)